CRM/Custom/Form/Field.php

```diff
@@ -507,7 +507,7 @@ public function buildQuickForm() {
     );
 
     // is active ?
-    $this->add('checkbox', 'is_active', ts('Active?'));
+    $this->add('advcheckbox', 'is_active', ts('Active?'));
 
     // is active ?
     $this->add('checkbox', 'is_view', ts('View Only?'));
```

Comment:
The respective fields have their default set to FALSE - hence update works fine. But good idea to do it anyway :)

Will update the PR.